tox.ini

```diff
@@ -22,6 +22,7 @@ setenv=
   ANSIBLE_ACTION_PLUGINS = {toxinidir}/plugins/actions
   ANSIBLE_CALLBACK_PLUGINS = {toxinidir}/plugins/callback
   ANSIBLE_CALLBACK_WHITELIST = profile_tasks
+  ANSIBLE_FILTER_PLUGINS = {toxinidir}/plugins/filter
```

Comment:
There's also tox-update.ini and tox-podman.ini files to update even if I'm not sure this is useful since we already define the ansible.cfg via ANSIBLE_CONFIG environment variable

Reply:
Done.